frontend/src/pages/detail/components/SectionTitle.tsx

```diff
@@ -0,0 +1,16 @@
+import * as S from "./SectionTitle.styled";
+
+interface SectionTitleProps {
+  text: string;
+}
```

Comment:
```suggestion
interface SectionTitleProps {
  title: string;
}
```

컴포넌트 이름도 `SectionTitle`이기 때문에 props명도 `title`이 더 자연스럽다고 생각하는데 아더의 생각은 어떠신가요??

Reply:
컴포넌트명 따라서 props 유동적으로 정하는게 더 자연스러운 거 같네요!